<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoControllerForm</class>
 <widget class="GUIFormBase" name="VideoControllerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="toolButtonRunMovie">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>再生・一時停止</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
   <property name="icon">
    <iconset resource="VideoController.qrc">
     <normaloff>:/Resources/Normal/Resources/Normal/PlayN.png</normaloff>
     <normalon>:/Resources/Select/Resources/Select/PauseS.png</normalon>:/Resources/Normal/Resources/Normal/PlayN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonForwardMovie">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>41</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>早送り</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="VideoController.qrc">
     <normaloff>:/Resources/Normal/Resources/Normal/ForwardN.png</normaloff>
     <normalon>:/Resources/Select/Resources/Select/ForwardS.png</normalon>:/Resources/Normal/Resources/Normal/ForwardN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderMovie">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonBackwardMovie">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>巻き戻し</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="VideoController.qrc">
     <normaloff>:/Resources/Normal/Resources/Normal/RewindN.png</normaloff>
     <disabledoff>:/Resources/Select/Resources/Select/RewindS.png</disabledoff>:/Resources/Normal/Resources/Normal/RewindN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonOpenFile">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="VideoController.qrc"/>
 </resources>
 <connections/>
</ui>
